### PR TITLE
MEN-4395: Bump version to 2021-01-11

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Review by lluiscampos — COMMENTED** [review]
Unsure about this one, I think we should wait for the release.

You might think that updating only master is innocent, but there is a good chance that we merge to `hosted` branch and then they become defacto released. 

At least I can say that in the past we only update these links during a release. I think this time we have made it in advance (because we knew there was an issue) but still we should wait for the docs until the release.

**Comment by oleorhagen** [comment]
So should I, as mentioned in the other PR:
* Do a publish job of the other client versions with the new image, but leave the docs untouched
* And leave the  ticket in the Merged state?

**Comment by lluiscampos** [comment]
> So should I, as mentioned in the other PR:
> 
>     * Do a publish job of the other client versions with the new image, but leave the docs untouched
> 
>     * And leave the  ticket in the Merged state?

Ticket is done, as the code has landed in its respective branches. And you don't need to publish the new images: release is not part of every single task, right?

**Comment by oleorhagen** [comment]
Holding off for the release

File: 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md [hunk 1]
```diff
@@ -34,8 +34,10 @@ Download the Raspberry Pi OS image with Mender integrated:
   * Download link: [Raspberry Pi 4 Model B][raspios-buster-lite-raspberrypi4-mender.img.xz]
 
 <!--AUTOVERSION: "mender-%.img.xz"/mender-convert-client -->
-[raspios-buster-lite-raspberrypi3-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2020-05-27-raspios-buster-lite-armhf/arm/2020-05-27-raspios-buster-lite-armhf-raspberrypi3-mender-master.img.xz
-[raspios-buster-lite-raspberrypi4-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2020-05-27-raspios-buster-lite-armhf/arm/2020-05-27-raspios-buster-lite-armhf-raspberrypi4-mender-master.img.xz
+[raspios-buster-lite-raspberrypi3-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2021-01-11-raspios-buster-armhf-lite/arm/2021-01-11-raspios-buster-armhf-lite-raspberrypi3-mender-latest.img.xz
+
```
Comment (lluiscampos) [inline 01.Get-started/01.Preparation/01.Prepare-a-Raspberry-Pi-device/docs.md:38]: Why this newline?

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**